setup.py

```diff
@@ -52,12 +52,13 @@ def command_exists(cmd):
 # Building wheel for distribution, update version file
 if 'MII_BUILD_STRING' in os.environ:
     # Build string env specified, probably building for distribution
-    with open('build.txt', 'w') as fd:
+    BUILD_FILE = 'build.txt'
```

Comment:
BUILD_FILE declaration needs to be moved outside the `if 'MII_BUILD_STRING'` statement

Reply:
I moved MII_BUILD_STRING, BUILD_FILE. I also get the environmental variable at the start and then do the validation immediately thereafter.